internal/scti/handlers.go

```diff
@@ -68,48 +68,48 @@ var (
 	lastSCTTimestamp *prometheus.GaugeVec     // origin => value
 	reqsCounter      *prometheus.CounterVec   // origin, op => value
 	rspsCounter      *prometheus.CounterVec   // origin, op, code => value
-	rspLatency       *prometheus.HistogramVec // origin, op, code => value
+	rspDuration      *prometheus.HistogramVec // origin, op, code => value
 )
 
 // setupMetrics initializes all the exported metrics.
 func setupMetrics() {
 	// TODO(phboneff): add metrics for deduplication and chain storage.
 	knownLogs = promauto.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "known_logs",
+			Name: "ct_known_logs_info",
 			Help: "Set to 1 for known logs",
 		},
 		[]string{"origin"})
 	lastSCTTimestamp = promauto.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "last_sct_timestamp",
-			Help: "Time of last SCT in ms since epoch",
+			Name: "ct_sct_last_timestamp_seconds",
```

Comment:
RFC6962 specifies `timestamp` in milliseconds, so it feels a bit unusual to export a metric in fractional seconds.
I'm sure there's a good reason, so it would be worth a comment somewhere in here explaining why we're doing that rather than exporting the value in its natural form.

Reply:
Same here, I forgot to link to https://prometheus.io/docs/practices/naming/, I've now added it. As explained in the PR overview comment, I'm on the fence as well, but I'm following best practices. I can easily be convinced that we should not follow that recommendation because of the RFC :D. I thought I'd give it a try though since in practice, I could not justify to myself that having milliseconds would actually be useful, other than matching with the RFC. This metric is likely to be used for two things:
 - graphs: either we just want to see a graph going upright, or we'll change the axis to be human parsable with a human readable time representation
 - alerting: if any it's unlikely we create alerts at the millisecond level